storage/postgres/where_clause_tree.go

```diff
@@ -99,10 +99,12 @@ func criterionSQL(c query.Criterion, dbTags []tagType, tableAlias string) (strin
 
 func buildRightOp(operator query.Operator, rightOp []string) (string, interface{}) {
 	rightOpBindVar := "?"
-	var rhs interface{} = rightOp[0]
+	var rhs interface{}
 	if operator.Type() == query.MultivariateOperator {
 		rightOpBindVar = "(?)"
 		rhs = rightOp
+	} else {
+		rhs = rightOp[0]
 	}
 	return rightOpBindVar, rhs
```

Comment:
This will return a positional argument with empty value, not sure how this will behave in the database.
What do you think about forbidding an empty array as rightOp on multivariate operators when the criteria is validated.